doc/production-hosting.md

```diff
@@ -36,38 +36,7 @@
 
 There are [too many bottlenecks][issue 2577]!
 
-## Badge servers
-
-There are three public badge servers on OVH VPS’s.
-
-| Cname                       | Hostname             | Type | IP             | Location           |
-| --------------------------- | -------------------- | ---- | -------------- | ------------------ |
-| [s0.servers.shields.io][s0] | vps71670.vps.ovh.ca  | VPS  | 192.99.59.72   | Quebec, Canada     |
-| [s1.servers.shields.io][s1] | vps244529.ovh.net    | VPS  | 51.254.114.150 | Gravelines, France |
-| [s2.servers.shields.io][s2] | vps117870.vps.ovh.ca | VPS  | 149.56.96.133  | Quebec, Canada     |
```

Comment:
We're still using these as proxying servers so we probably should keep them in the docs, somewhere, until they are no longer part of the production setup.

Reply:
Agreed, especially since this is apparently a point of reference for some vendors currently whitelisting our traffic